src/lib/util/validation.js

```diff
@@ -11,6 +12,10 @@ export const uniqueConstraintError = () => ({
   code: UNIQUE_CONSTRAINT,
 });
 
+export const invalidSymbolError = () => ({
+  code: INVALID_SYMBOL,
+});
+
```

Comment:
The design for these validation errors was to allow including additional data in the validation error alongside the error type. In this case we could include the offending characters and use that to render a more detailed message.

```js
export const invalidSymbolError = (characters: string) => ({
  code: INVALID_SYMBOL,
  character,
});
```
```js
const messageMap = {
  [INVALID_SYMBOL]: ({character}) => `Cannot contain ${character}`,
};
```